Test/Machine/test_machine.py

```diff
@@ -63,6 +64,10 @@ def initializer(rng, shape):
         dtype=complex,
     )
 
+    dm_machines["Jax NDM"] = nk.machine.density_matrix.JaxNdmSpinPhase(
```

Comment:
I was thinking that maybe we should just call this NdmSpinPhase and remove Jax... ?

Reply:
Since it's really a Jax machine, which is an optional dependency, and maybe in 2025 when I'll be old i'll write a Torch version, I'd leave this as is for now.